docs/src/model_setup/boundary_conditions.md

```diff
@@ -92,8 +92,8 @@ julia> model.velocities.v.boundary_conditions
 Oceananigans.FieldBoundaryConditions, with boundary conditions
 ├── west: PeriodicBoundaryCondition
 ├── east: PeriodicBoundaryCondition
-├── south: OpenBoundaryCondition{Nothing}: Nothing
-├── north: OpenBoundaryCondition{Nothing}: Nothing
+├── south: OpenBoundaryCondition{Nothing}: 0.0
+├── north: OpenBoundaryCondition{Nothing}: 0.0
```

Comment:
what does this mean exactly? The type is `Nothing` but the value is `0.0` somehow?

Reply:
The type is the matching scheme type (and shows like that because `show` isn't defined for `Open`)

Reply:
Why are we changing the value of the default?

We can't use `0.0` because this is Float64. We can use `0` (integer) if we need a default number. `Nothing` was the original value though which seems best.

Reply:
Oh I see I missed that, I can change it back to nothing

Reply:
this isnt' changed yet